tests/dynamic-attribute-list.spec.js

```diff
@@ -0,0 +1,79 @@
+const {
+    expect
+} = require("chai");
+const fs = require("fs");
+const nock = require("nock");
+const uuid = require("uuid");
+
+const config = require("../config");
+const {
+    DynamicAttributeListRequest,
+    AttributeList,
+    getDynamicAttributeList,
+} = require("../src/dynamic_attribute_list_service");
+const Payload = require('../src/request/payload').Payload;
+
+const privateKeyFile = fs.readFileSync('./tests/sample-data/keys/node-sdk-test.pem', 'utf8');
+
+const APP_ID = uuid();
+
+describe("dynamicAttributeList", () => {
+    const callbackURL = "/login/f842b2f6-b41f-45ba-8a53-a18d40379b82";
+    const wantedAttributeList = [{
+        "name": "full_name",
+        "derivation": "",
+        "optional": "false",
+    }];
+    const wantedRememberMe = false;
+    const attributeList = AttributeList(
+        wantedAttributeList,
+        null,
+        wantedRememberMe,
+    );
+    const dynamicAttributeListRequest = new DynamicAttributeListRequest(callbackURL, attributeList);
+    //const QRCODE_LINK = "https://staging0.code.yoti.com/CAEaJDRjNTQ3M2IxLTNiNzktNDg3My1iMmM4LThiMTQxZDYwMjM5ODAC";
```

Comment:
Could we remove this? We can put it in the wiki for reference.

If we want ease of use when devs are working on it, we can add it as an environment variable